config/env_spec_request.go

```diff
@@ -524,6 +525,20 @@ func (e *EnvironmentSpecRequest) GetHTTPRequestTransforms() (transforms HTTPRequ
 	return transforms
 }
 
+func (e *EnvironmentSpecRequest) GetDynamicMetadata() (metadata map[string]interface{}) {
+	if e != nil {
+		op := e.GetOperation()
+		if op != nil && len(op.DynamicMetadata) > 0 {
```

Comment:
Because this is a map, when absent from the yaml, it may actually nil. Therefore, if users actually specify an empty map in an operation:
```
...
dynamic_metadata: {}
...
```
should we regard that as overriding the API level settings?

Reply:
This wouldn't handle the case of an empty map at the op level, no?